04-Intro_to_Vue/assignment/css/_settings.responsive.scss

```diff
@@ -0,0 +1,52 @@
+// Media Query Generator
+
+@mixin media-query($mq) {
+
+    $breakpoint-found: false;
+
+    // Loop through the list of breakpoints we’ve provided in our settings file.
+    @each $breakpoint in $breakpoints {
```

Comment:
nice, but this is vey much overkill. :)

But also brings to mind one of my favorite quotes.....

"anything worth doing is worth overdoing"